src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -61,9 +61,14 @@ trait ClassLikeSupport:
       )
 
   extension (c: ClassDef):
+      def nonprivateMembers = c.body.filterNot(_.symbol.isHiddenByVisibility)
```

Comment:
The name suggests that only private members are filtered out, can we instead name this `membersToDocument` (as that seems to be the intention here)?

Reply:
done